### PR TITLE
[exporter/jobs] account metrics per job state

### PR DESCRIPTION
**Pull request by abhinavDhulipala**
Denorm account state per job state. Due to a renamed metric, this is a **breaking** change and must be accompanied by a upgraded Grafana dashboard

### CODE REVIEW

File: exporter/jobs.go [hunk 1]
```diff
@@ -306,19 +308,19 @@ func NewJobsController(config *Config) *JobsCollector {
...
-		jobScrapeDuration:      prometheus.NewDesc("slurm_job_scrape_duration", fmt.Sprintf("how long the cmd %v took (ms)", cliOpts.squeue), nil, nil),
+		jobAllocCpus:            prometheus.NewDesc("slurm_job_alloc_cpus", "amount of cpus allocated per job", []string{"jobid"}, nil),
+		jobAllocMem:             prometheus.NewDesc("slurm_job_alloc_mem", "amount of mem allocated per job", []string{"jobid"}, nil),
+		userJobStateTotal:       prometheus.NewDesc("slurm_user_state_total", "total jobs per state per user", []string{"username", "state"}, nil),
+		userJobMemAlloc:         prometheus.NewDesc("slurm_user_mem_alloc", "total mem alloc per user", []string{"username", "state"}, nil),
+		userJobCpuAlloc:         prometheus.NewDesc("slurm_user_cpu_alloc", "total cpu alloc per user", []string{"username", "state"}, nil),
+		partitionJobStateTotal:  prometheus.NewDesc("slurm_partition_job_state_total", "total jobs per partition per state", []string{"partition", "state"}, nil),
+		accountJobStateMemAlloc: prometheus.NewDesc("slurm_account_mem_state_alloc", "alloc mem consumed per account per job state", []string{"account", "state"}, nil),
```
Comment (ghost) [inline exporter/jobs.go]: I think this should be `slurm_account_job_state_mem_alloc`

File: exporter/jobs.go [hunk 2]
```diff
@@ -306,19 +308,19 @@ func NewJobsController(config *Config) *JobsCollector {
...
+		accountJobStateCpuAlloc: prometheus.NewDesc("slurm_account_cpu_state_alloc", "alloc cpu consumed per account per job state", []string{"account", "state"}, nil),
```
Comment (ghost) [inline exporter/jobs.go]: I think this should be `slurm_account_job_state_cpu_alloc`

Comment (abhinavDhulipala) [reply exporter/jobs.go]: 7

File: exporter/jobs.go (hunk 1 above)
Comment (abhinavDhulipala) [reply exporter/jobs.go]: Addressed

**Pull request merged**